src/components/AssetPreview.tsx

```diff
@@ -32,11 +33,9 @@ export function AssetPreview({ previewUri, metadata }: Props): ReactElement {
     <Box mb={0.25}>
       <Box bgcolor="background.paper">
         <Grid container direction="row">
-          {previewUri ? (
-            <FitImage maxWidth="250px" maxHeight="175px" alt="Upload Preview" src={previewUri} />
-          ) : (
-            <AssetIcon icon={previewComponent} />
-          )}
+          <div style={{ width: PREVIEW_DIMENSIONS.maxWidth, height: PREVIEW_DIMENSIONS.maxHeight }}>
+            {previewUri ? <FitImage alt="Upload Preview" src={previewUri} /> : <AssetIcon icon={previewComponent} />}
+          </div>
```

Comment:
Should this wrapper `div` be part of the `FitImage` component?

As I understand, the functionality is the same just with a better approach (parent + scale-down object-fit), but it is not enclosed in a single component, and the `FitImage` is no longer true to its name as well in the current state 🙂

Reply:
Ah I see it is more tricky because we also display the AssetIcon sometime.

Reply:
So I actually decided it is best to always have this preview to be always the same size and only scale down the image inside the preview if needed (and for smaller images to not stretch them). This IMHO makes more sense because the layout always stays the same. But You are right that I should review how `AssetIcon` component works and maybe only need to size correctly the `FitImage`.

Reply:
I edited it to now always have the same width and height (if there is an image preview or just icon).